scripts/kpm.cmd

```diff
@@ -2,7 +2,7 @@
 SETLOCAL
 SET ERRORLEVEL=
 
-"%~dp0klr" --appbase "%CD%" %K_OPTIONS% --lib "%~dp0lib\Microsoft.Framework.PackageManager" Microsoft.Framework.PackageManager %*
+"%~dp0dotnet" --appbase "%CD%" %K_OPTIONS% --lib "%~dp0lib\Microsoft.Framework.PackageManager" Microsoft.Framework.PackageManager %*
```

Comment:
odd to use `K_OPTIONS` in the `kpm` script.  should this be `DOTNET_OPTIONS` since the env variable seems to be used for all `dotnet` invocations?